gubernator/view_build_test.py

```diff
@@ -59,6 +59,8 @@ def test_bad_xml(self):
 
 
 class BuildTest(main_test.TestBase):
+    # pylint: disable=too-many-public-methods
```

Comment:
aside: I would have expected pylint to special-case test classes for this sort of thing...